tests/flink-it/src/test/scala/org/apache/celeborn/tests/flink/HybridShuffleWordCountTest.scala

```diff
@@ -144,10 +137,9 @@ class HybridShuffleWordCountTest extends AnyFunSuite with Logging with MiniClust
     configuration.setString("rest.bind-port", "8081-8099")
     val env = getEnvironment(configuration);
     env.getConfig.setParallelism(parallelism)
-    env.setRestartStrategy(RestartStrategies.fixedDelayRestart(1, 0L))
     env.disableOperatorChaining()
     // make parameters available in the web interface
-    WordCountHelper.execute(env, parallelism)
+    WordCountHelper.execute(env, 1)
```

Comment:
Add comment for this line of change. If we bump flink to 2.1, this can be reverted IMO.